AutoGVP/01-annotate_variants_CAVATICA_input.R

```diff
@@ -171,7 +171,23 @@ clinvar_anno_vcf_df <- address_conflicting_intrep(clinvar_anno_vcf_df)
 
 
 ## get latest calls from submission files
-submission_summary_df <- vroom(input_summary_submission_file,
+submission_info_df <- vroom(input_variant_summary,
+  delim = "\t",
+  col_types = c(ReferenceAlleleVCF = "c", AlternateAlleleVCF = "c", PositionVCF = "i", VariationID = "n"),
+  show_col_types = FALSE
+) %>%
+  # add vcf id column
+  dplyr::mutate(
+    vcf_id = str_remove_all(paste(Chromosome, "-", PositionVCF, "-", ReferenceAlleleVCF, "-", AlternateAlleleVCF), " "),
+    vcf_id = str_replace_all(vcf_id, "chr", ""),
+    VariationID = as.double(noquote(VariationID))
+  ) %>%
+  group_by(VariationID) %>%
+  arrange(LastEvaluated) %>%
```

Comment:
```suggestion
  arrange(ymd(LastEvaluated)) %>%
```
think you want this?

not a huge deal, but I might think that arranging in order of most recent would be better so in this case:

```r
  arrange(desc(ymd(LastEvaluated))) %>%
 dplyr::slice_head(n = 1) %>%
```

in case you wanted to quickly pull up the df and not have to resort that column